src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -356,12 +309,15 @@ def cli_cosmosdb_sql_container_update(client,
                                           None):
         logger.debug('replacing SQL container')
 
+    sql_container_create_update_resource = SqlContainerCreateUpdateParameters(
+        resource=sql_container_resource,
+        options={})
```

Comment:
Why do you want to pass options with {} here?

Reply:
This is the same reasoning as with MongoDBCollectionCrateUpdateParameters. It is a required parameter in the SDK, but we do not allow option updates via this command.